content/applications/finance/fiscal_localizations/colombia.rst

```diff
@@ -83,43 +100,79 @@ Next, configure the :guilabel:`Fiscal Information` in the :guilabel:`Sales & Pur
+  - :guilabel:`Software PIN`: the PIN selected in the operation mode configuration in the |DIAN|
+    portal.
+  - :guilabel:`Testing ID`: the testing ID generated by |DIAN| and obtained after testing the
+    operation mode.
 
-Once Odoo and Carvajal are fully configured and ready for production, deactivate the :guilabel:`Test
-mode` checkbox to use the production database.
+- Configure the available :guilabel:`Certificates` to sign your electronic documents:
 
-.. image:: colombia/carvajal-configuration.png
-   :align: center
-   :alt: Configure credentials for Carvajal web service in Odoo.
+  - :guilabel:`Name`: the name of the certificate.
+  - :guilabel:`Certificate`: upload the certificate file in PEM format. After uploading a valid
+    certificate file, the following fields display:
 
-.. important::
-   :guilabel:`Test mode` must **only** be used on duplicated databases, **not** the production
-   environment.
+    - :guilabel:`Certificate Password`: the password to decrypt the PKS file.
+    - :guilabel:`Private Key`: select or create a new a private key.
+    - :guilabel:`Public Key`: Set a public key to be used if the key contained in the certificate
+      file is invalid.
+
+.. image:: colombia/dian-credentials-configuration.png
+   :alt: Colombian electronic invoicing credentials configured.
+
+.. note::
+   In a multi-company database, it is possible to have a different certificate for each company.
+
+.. _colombia/dian-environment-config:
+
+|DIAN| environment configuration
+--------------------------------
+
+The |DIAN| electronic invoicing module offers three different |DIAN| environments to connect with,
+each can be enabled from :menuselection:`Accounting --> Configuration --> Settings` and scroll to
+the :guilabel:`Colombian Electronic Invoicing` section:
+
+- **Certification environment**: This environment is useful to pass the |DIAN| certification
+  process and obtain the *Enabled* status to invoice from Odoo. To activate it, tick both the
+  :guilabel:`Test Environment` and the :guilabel:`Activate the certification process` checkboxes.
+
+- **Testing environment**: This environment allows reproducing electronic invoicing flows
+  and validations in the |DIAN| testing portal. To activate it, tick only the :guilabel:`Test
+  Environment` checkbox.
+
+- **Production environment**: Activate production databases to generate valid electronic documents.
+  To activate it, untick both the :guilabel:`Test Environment` and the :guilabel:`Activate the
+  certification process` checkboxes.
+
+.. seealso::
+   For electronic invoicing configurations using the Carvajal solution, review the following video:
+   `Configuración de Facturación Electrónica - Localización de Colombia
+   <https://www.youtube.com/watch?v=bzweMwTEbfY&list=PL1-aSABtP6ABxZshems3snMjx7bj_7ZsZ&index=3>`_.
 
 Report data configuration
 -------------------------
 
+.. Only for Carvajal config ⬇️
+
```

Comment:
In my testing, the "Report Configuration" is only visible in the **Colombian Electronic Invoicing** section when **Carvajal** is selected as the **Electronic Invoicing Provider**. Can you please double check this? :)

Reply:
@samueljlieber this is totally true! In fact this `Report data configuration` section should be deprecated completely. I will add "Tip" and possibly a screenshot in the `Company configuration` section. Since now these fields are used for the fiscal information of the PDF